src/pages/Customers/ListCustomers.js

```diff
@@ -1,7 +1,95 @@
-import React from "react";
+import React, { Fragment, useEffect, useState, useCallback } from "react";
+import Navbar from "common/Navbar";
+import { FormattedMessage } from "react-intl";
+import { Link } from "react-router-dom";
+import withWrapper from "common/withWrapper";
+import { useNotification } from "notification";
 
 const ListCustomers = () => {
-    return <div>I am list customers page</div>;
+    const [tableData, setTableData] = useState([]);
+    useEffect(() => {
+        fetchData();
+    }, []);
+    const { triggerNotification } = useNotification();
+
+    // Function to fetch data from local storage
+    const fetchData = useCallback(() => {
+        let customerData = [];
+        if (localStorage.getItem("customer_data")) {
+            try {
+                customerData = JSON.parse(
+                    localStorage.getItem("customer_data")
+                );
+            } catch (e) {
+                triggerNotification("Failed parsing customer data", {
+                    type: "error",
+                });
+                localStorage.removeItem("customer_data");
+            }
+        }
+        setTableData(customerData);
+    }, [tableData]);
```

Comment:
you are not consuming `tableData` in this function

Reply:
Fixed it